scripts/validation/validate.js

```diff
@@ -143,3 +154,11 @@ function assertValid(schema, subject) {
     throw new Error(errorMessage);
   }
 }
+
+async function getModifiedServices() {
+  const git = simpleGit(path.resolve(__dirname, '../..'), { maxConcurrentProcesses: 1 });
+  const modifiedFiles = await (await git.diff([ '--name-only', 'master...' ])).split('\n');
```

Comment:
Why use triple-dot notation? From [what I understand](https://stackoverflow.com/questions/462974/what-are-the-differences-between-double-dot-and-triple-dot-in-git-com#24186641), this means we will include in the diff all services that have changed in `master` since the branch we're on has diverged, which is not needed.
```suggestion
  const modifiedFiles = await (await git.diff([ '--name-only', 'master' ])).split('\n');
```

Reply:
From my tests, it's exactly the opposite, that's why I added it :)

Reply:
Be careful, `git log` and `git diff` behave differently with `...` and the link you provided is for `git log` but we are using `git diff` here. See https://stackoverflow.com/a/7256391

Reply:
I provided in https://github.com/ambanum/CGUs/pull/110#discussion_r484880144 the way I tested expected behaviour.